database/team.js

```diff
@@ -1,17 +1,32 @@
-import { db } from './database.js';
+import { db } from "./database.js";
 
 export async function loadTeams() {
-    return await db.all('SELECT id, discord_snowflake FROM team');
+  return await db.all("SELECT id, discord_snowflake FROM team");
 }
 
 export async function loadActiveTeams() {
-    return await db.all('SELECT id, discord_snowflake FROM team WHERE active = 1');
+  return await db.all(
+    "SELECT id, discord_snowflake FROM team WHERE active = 1"
+  );
 }
 
 export async function loadTeam(teamId) {
-    return await db.get('SELECT id, discord_snowflake FROM team WHERE id = ?', teamId);
+  return await db.get(
+    "SELECT id, discord_snowflake FROM team WHERE id = ?",
+    teamId
+  );
 }
 
 export async function loadTeamFromSnowflake(snowflake) {
-    return await db.get('SELECT id, discord_snowflake FROM team WHERE discord_snowflake = ?', snowflake);
-}
\ No newline at end of file
+  return await db.get(
+    "SELECT id, discord_snowflake FROM team WHERE discord_snowflake = ?",
+    snowflake
+  );
+}
+
+export async function loadTeamData(snowflake) {
```

Comment:
ideally this will eventually be housed in roster.js which is where I want to put this historical-type data but that's just an FYI.